### PR TITLE
IZPACK-1655 - Allow exclude of properties that may contain passwords

### PR DESCRIPTION
**Pull request by mezmerize**
IZPACK-1655 - added option to exclude password properties.

### CODE REVIEW

**Comment by reinhapa** [comment]
@akuhtz could you take a look at this too?

File: izpack-maven-plugin/src/main/java/org/izpack/mojo/IzPackNewMojo.java [hunk 1]
```diff
@@ -240,13 +247,16 @@ private void initMavenProperties(PropertyManager propertyManager)
                 } else {
                     value = properties.getProperty(propertyName);
                 }
-                if (propertyManager.addProperty(propertyName, value))
-                {
-                    getLog().debug("Maven property added: " + propertyName + "=" + value);
-                }
-                else
+                if (!propertyName.contains("password") || includePasswordProperties)
```
Comment (akuhtz) [inline izpack-maven-plugin/src/main/java/org/izpack/mojo/IzPackNewMojo.java]: There might be others like `passphrase` that will not be skipped here. Providing a regex check could be more flexible here.
But it matches the requirement of the current issue.

Comment (rkrell) [reply izpack-maven-plugin/src/main/java/org/izpack/mojo/IzPackNewMojo.java]: I don't like this kind of static substring paring, neither. Should be solved differently, for instance by XML attributes.

**Comment by mezmerize** [comment]
Based on your comments I have changed it to a list of properties that will be marked for exclusion.

**Comment by rkrell** [comment]
> Based on your comments I have changed it to a list of properties that will be marked for exclusion.

This seems fine. @reinhapa , @akuhtz and others, how about that?

**Comment by reinhapa** [comment]
Looks good to me.

**Comment by akuhtz** [comment]
Yes. Thank you @mezmerize .

**Pull request merged**